hardware_interface/test/test_component_parser.cpp

```diff
@@ -910,7 +927,7 @@ TEST_F(TestComponentParser, successfully_parse_valid_urdf_actuator_modular_robot
   EXPECT_EQ(hardware_info.joints[0].class_type, "ros2_control_components/PositionJoint");
   ASSERT_THAT(hardware_info.joints[0].command_interfaces, SizeIs(0));
   ASSERT_THAT(hardware_info.joints[0].state_interfaces, SizeIs(1));
-  EXPECT_EQ(hardware_info.joints[0].state_interfaces[0], "position");
+  EXPECT_EQ(hardware_info.joints[0].state_interfaces[0].name, "position");
   ASSERT_THAT(hardware_info.joints[0].parameters, SizeIs(2));
   EXPECT_EQ(hardware_info.joints[0].parameters.at("max_position_value"), "${PI}");
```

Comment:
We should also change those into "min" and "max"

Reply:
If I understand correctly, the key of `joints[0].parameters` can be any string. I believe you are thinking of `joints[0].command_interfaces[0].min` and `joints[0].command_interfaces[0].min`

For example, this is perfectly valid:

`EXPECT_EQ(hardware_info.joints[0].parameters.at("serial_number"), "A12345C63");`

Reply:
A lot of the example URDF has this:

```
    <joint name="joint1">
      <classType>ros2_control_components/PositionJoint</classType>
      <param name="min_position_value">-1</param>
      <param name="max_position_value">1</param>
    </joint>
```

Maybe these examples are deprecated and they need to be deleted? Hopefully in a follow-up PR?

Reply:
Ok, than leave it as it is.

Reply:
Let us see than later, what we do.

Probably this Example URDFs should go into test_robot_hatdware package.